scripts/sct_analyze_lesion.py

```diff
@@ -71,13 +66,13 @@ def get_parser():
         default=None,
         required=False)
     optional.add_argument(
-        "-f",
-        help="Path to folder containing the atlas/template registered to the anatomical image. If provided, the "
-             "function computes: (i) the distribution of each lesion depending on each vertebral level and on each"
+        "-t",
```

Comment:
Old version: template analysis was not mandatory --> in that case the output was simply "lesion count, volume, length".

The template allows to compute statistics per tracts / levels.

@jcohenadad: Do we want to make this template analysis mandatory? It does make the processing longer. If not, do you have a suggestion of parser_parameter to prevent the template analysis?

Reply:
how about a flag "-no-atlas"?
p.s. we are slowly moving towards more verbose flags (discussion for next SCT meeting)

Reply:
actually, after thinking about it, we could also make "-t" optional (which is already the case). When not declared, there is no atlas analysis. When declared, there is. I find it more intuitive.

Reply:
Sounds good!